ci/requirements/linux_min_deps_conda.txt

```diff
@@ -10,4 +10,5 @@ pytest-sugar
 scipy
 networkx
 hsluv
-kiwisolver
\ No newline at end of file
+kiwisolver
+sphinx-gallery
```

Comment:
SG should not be in the minimal deps, why did you have to add it? If it's for the tests I would just add this to the test file:
```
try:
    import sphinx_gallery
except Exception:
    pytest.skip("skipping sphinx-gallery tests", allow_module_level=True)
```
you can do something similar with decorators if you want

Reply:
Yes it was for the tests. I guess that's a good point. Just because it is a test doesn't mean it needs to be run in every environment.